phone.go

```diff
@@ -0,0 +1,68 @@
+package faker
+
+import (
+	"fmt"
+	"github.com/agoalofalife/faker/support/slice"
```

Comment:
This package, can confuse the package version manager. Change into
`github.com/bxcodec/faker/support/slice`

Reply:
wait. I think I will replaced it later. I'm afraid changing this will cause an error on travis

Reply:
Yes it was replaced before

Reply:
I have added a new feature, its now not in general repository

Reply:
Anymore addition ??, if not, I will merge this, and repair the packages..

Reply:
Yes, it is possible to correct only after the merge